styles/styles.css

```diff
@@ -51,6 +51,7 @@
   --font-weight-bold: 700;
 
   /* Headings */
+  --heading-font-size-xxxxl: 50px;
```

Comment:
Shrink this down to xxxl; 2px shouldn't be that big of a deal for the change in experience.